eventmesh-connectors/eventmesh-connector-rocketmq/src/main/java/org/apache/eventmesh/connector/rocketmq/source/connector/RocketMQSourceConnector.java

```diff
@@ -273,7 +273,7 @@ private long removeMessage(MessageQueue mq, long offset) {
 
     private void execScheduleTask() {
         commitOffsetScheduleService.scheduleAtFixedRate(this::commitOffsetSchedule, sourceConfig.connectorConfig.getCommitOffsetIntervalMs(),
-            sourceConfig.connectorConfig.getCommitOffsetIntervalMs(), TimeUnit.MILLISECONDS);
+                sourceConfig.connectorConfig.getCommitOffsetIntervalMs(), TimeUnit.MILLISECONDS);
     }
```

Comment:
additional indentation.

Reply:
This is because I excluded this file from the spotless conversion list first, and fixed the indentation issue in a later commit.
I'll fix this in a new round of commits.